src/ui/component/skeleton/skeleton.tsx

```diff
@@ -0,0 +1,52 @@
+import type { CSSProperties, FC } from 'react'
+import classNames from 'classnames'
+import './skeleton.scss'
+
+type SkeletonProps = {
+  width?: number | string
+  height?: number | string
+  fontSize?: number | string
+  variant?: 'text' | 'rectangular' | 'rounded' | 'circular'
+  className?: string
+  style?: CSSProperties
+  count?: number
+}
+
+export const Skeleton: FC<SkeletonProps> = ({
+  width,
+  height,
+  variant = 'text',
```

Comment:
What's the purpose of the `text` variant and the `fontSize` prop?
Please, stay KISS, just use a rounded variant with a height property to customize the skeleton.

Reply:
@fredericvilcot  `text` variant apply default `height` or `border-radius` properties:
```css
&.text {
  height: 1.2em;
  border-radius: 4px;
 }
```
It's an opinionated variant based on what Material-UI does.
We can make the API more simple by providing a `radius` property, what do you think?
It handles easily all our variant cases, except the rounded one, that must be computed manually.

The `fontSize` is not relevant, we can definitely remove it.

Reply:
@0xLodz 
I'll suggest to remove the `fontSize` property.
About the `text` variant, IMHO it brings nothing more than a rounded variant with a `height` property and a different radius.
We can keep it for now, but we must keep focusing on small components that do the job for now.